src/utils/node/pm2.ts

```diff
@@ -10,6 +11,10 @@ import {
 	stop,
 } from 'pm2';
 
+export interface Pm2Env {
```

Comment:
it seems like original `ProcessDescription` has definition of Pm2Env, but why create new one?

Reply:
it's not exported, so when I try `const { pm2_env: { pm_cwd } } = await describeApplication(name);` it throws error unknow type. so i defined this type.